src/main/webapp/script.js

```diff
@@ -15,14 +15,6 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function userAuthorization() {
```

Comment:
Small nit: This function name would be more clear if it's phrased as an action that describes what it's doing. I'd recommend renaming this to something like `authorizeUser`.